target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -51,6 +51,16 @@ comfast,cf-e120a-v3)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e313ac)
+	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
```

Comment:
Convert to default trigger in DTS, as in
https://github.com/openwrt/openwrt/commit/abbbecaa73dcb5a36932a224f74488d0c5e3b3cd#diff-285ac400812ddee4d019c28256ada29dR70

Reply:
Done.